client/src/pages/FilmDetails.tsx

```diff
@@ -24,7 +28,32 @@ export function FilmDetailPage() {
       }
     }
     getDetails();
-  }, [filmId]);
+
+    async function getWishlist() {
+      if (user) {
```

Comment:
This is one of those cases where I would "fail fast" so that the code is easier to read. When I first looked at this, I had a hard time figuring out where the end of the `if (user)` block was.